docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -0,0 +1,256 @@
+
+    Creating schema: GLOBALEVENTID:String,Actor1Name:String,Actor1CountryCode:String,Actor2Name:String,Actor2CountryCode:String,EventCode:String,NumMentions:Integer,NumSources:Integer,NumArticles:Integer,ActionGeo_Type:Integer,ActionGeo_FullName:String,ActionGeo_CountryCode:String,dtg:Date,geom:Point:srid=4326
+
+    Generating test data
+
+    Writing test data
+    Wrote 2356 features
+
+    Running test queries
+    Running query BBOX(geom, -120.0,30.0,-75.0,55.0) AND dtg DURING 2017-12-31T00:00:00+00:00/2018-01-02T00:00:00+00:00
+    01 719024913=719024913|||VICTORIA|AUS|173|2|1|2|4|Victoria Harbour, Ontario, Canada|CA|2018-01-01T00:00:00.000Z|POINT (-79.7667 44.75)
+    02 719024913=719024913|||VICTORIA|AUS|173|2|1|2|4|Victoria Harbour, Ontario, Canada|CA|2018-01-01T00:00:00.000Z|POINT (-79.7667 44.75)
+    03 719024918=719024918|||EMPLOYER||020|10|1|10|3|Brooklyn, New York, United States|US|2018-01-01T00:00:00.000Z|POINT (-75.1704 42.3442)
+    04 719024918=719024918|||EMPLOYER||020|10|1|10|3|Brooklyn, New York, United States|US|2018-01-01T00:00:00.000Z|POINT (-75.1704 42.3442)
+    05 719024923=719024923|||INDUSTRY||084|3|1|3|2|Pennsylvania, United States|US|2018-01-01T00:00:00.000Z|POINT (-77.264 40.5773)
+    06 719024923=719024923|||INDUSTRY||084|3|1|3|2|Pennsylvania, United States|US|2018-01-01T00:00:00.000Z|POINT (-77.264 40.5773)
+    07 719024924=719024924|||CORPORATION||115|2|1|2|4|Saskatoon, Saskatchewan, Canada|CA|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+    08 719024924=719024924|||CORPORATION||115|2|1|2|4|Saskatoon, Saskatchewan, Canada|CA|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+    09 719024925=719024925|||CORPORATION||172|2|1|2|4|Saskatoon, Saskatchewan, Canada|CA|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+    10 719024925=719024925|||CORPORATION||172|2|1|2|4|Saskatoon, Saskatchewan, Canada|CA|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+
+    Returned 1338 total features
+
+    Running query BBOX(geom, -120.0,30.0,-75.0,55.0) AND dtg DURING 2017-12-31T00:00:00+00:00/2018-01-02T00:00:00+00:00
+    Returning attributes [GLOBALEVENTID, dtg, geom]
+    01 719024913=719024913|2018-01-01T00:00:00.000Z|POINT (-79.7667 44.75)
+    02 719024913=719024913|2018-01-01T00:00:00.000Z|POINT (-79.7667 44.75)
+    03 719024918=719024918|2018-01-01T00:00:00.000Z|POINT (-75.1704 42.3442)
+    04 719024918=719024918|2018-01-01T00:00:00.000Z|POINT (-75.1704 42.3442)
+    05 719024923=719024923|2018-01-01T00:00:00.000Z|POINT (-77.264 40.5773)
+    06 719024923=719024923|2018-01-01T00:00:00.000Z|POINT (-77.264 40.5773)
+    07 719024924=719024924|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+    08 719024924=719024924|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+    09 719024925=719024925|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+    10 719024925=719024925|2018-01-01T00:00:00.000Z|POINT (-106.667 52.1333)
+
+    Returned 1338 total features
```

Comment:
hmm, this is way off - the other tutorials returned 669 results. we should look into this some... can you add a ticket for it?

Reply:
https://geomesa.atlassian.net/browse/GEOMESA-2183

Reply:
haha, nice resolution